### DISCUSSION TITLE
ergo pool recognized but mostly bad info

### DISCUSSION THREAD

**Discussion by def670** [coinfoundry/miningcore#977] [Q&A]
miningcore server up and able to mine on it but ergo node info is incorrect so i suspect i forked up somewhere.

Oct 20 00:44:33 ERGO Miningcore[27778]: Network Connected:      mainnet
Oct 20 00:44:33 ERGO Miningcore[27778]: Detected Reward Type:   POW
Oct 20 00:44:33 ERGO Miningcore[27778]: Current Block Height:   601642
Oct 20 00:44:33 ERGO Miningcore[27778]: Current Connect Peers:  0
Oct 20 00:44:33 ERGO Miningcore[27778]: Network Difficulty:     668095.9453222128
Oct 20 00:44:33 ERGO Miningcore[27778]: Network Hash Rate:      5.44 KH/s
Oct 20 00:44:33 ERGO Miningcore[27778]: Stratum Port(s):        3052
Oct 20 00:44:33 ERGO Miningcore[27778]: Pool Fee:               0.5%

ergo node is showing 45 peers connected - blockchain height is proper - difficulty and network hashrate incorrect

im using 
If any other info is needed just ask - ill get it here as quick as possible
thanks

**Comment by vinnielima** [comment]
Did you get any information on this?  Trying to look for sample Ergo config.json.

**Reply by def670** [reply]
https://github.com/coinfoundry/miningcore/discussions/895 there is a eg ergo config.json there...sorry i have not progressed any with this yet - too busy with RL

**Reply by vinnielima** [reply]
Yep, pieced it together.  What I was missing is the daemon entry should point to your ergo node. Thanks.
